pkg/operator/csi/csidrivercontrollerservicecontroller/helpers.go

```diff
@@ -63,3 +65,23 @@ func WithSecretHashAnnotationHook(
 		return nil
 	}
 }
+
+// WithReplicasHook sets the deployment.Spec.Replicas field according to the number
+// of available nodes. The number of nodes is determined by the node selector
+// specified in the field deployment.Spec.Templates.NodeSelector.
```

Comment:
Please add a note that MaxSurge should be set in the Deployment when node ports or hostNetwork are used.

Reply:
Done

Reply:
Done